src/main/xsl/preprocess/maprefImpl.xsl

```diff
@@ -30,17 +30,51 @@ See the accompanying LICENSE file for applicable license.
   <xsl:key name="reltable" match="//*[contains(@class, ' map/topicref ')]" use="@dita-ot:orig-format"/>
 
   <xsl:template match="@* | node()">
+    <xsl:param name="relative-path" as="xs:string">#none#</xsl:param>
```

Comment:
We really should refactor the code to stop using these `#none#` magic strings for null values and just start supporting `()` instead.

Reply:
Agreed, but probably not a good idea to do in the hotfix...